packages/ui/src/components/UnitInput/__stories__/Playground.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Decorator } from '@storybook/react'
+import { Template } from './Template.stories'
+
+export const Playground = Template.bind({})
+
+Playground.decorators = [
+  Story => (
+    <div>
+      <div style={{ display: 'inline-flex' }}>
```

Comment:
Can you add a `height: '200px'` or more in order to see correctly the select input part in storybook?

<img width="1038" alt="Screenshot 2024-05-02 at 10 01 10" src="https://github.com/scaleway/ultraviolet/assets/15812968/ab1c7d6c-030c-4919-a8e9-6fd2e6bd9100">